package/python-kiwi-keg.spec

```diff
@@ -12,62 +12,72 @@
 # case the license is the MIT License). An "Open Source License" is a
 # license that conforms to the Open Source Definition (Version 1.9)
 # published by the Open Source Initiative.
-
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
 
-%if 0%{?suse_version} > 1500
-%bcond_without libalternatives
+%if 0%{?suse_version} >= 1600
+%define pythons %{primary_python}
 %else
-%bcond_with libalternatives
+%{?sle15_python_module_pythons}
 %endif
 
-%define         skip_python2 1
-Name:           python-kiwi-keg
-Version:        2.1.1
-Release:        0
-URL:            https://github.com/SUSE-Enceladus/keg
-Summary:        KEG - Image Composition Tool
-License:        GPL-3.0-or-later
-Source:         keg-%{version}.tar.gz
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-BuildRequires:  %{python_module Jinja2}
-BuildRequires:  %{python_module Sphinx}
-BuildRequires:  %{python_module base >= 3.6}
-BuildRequires:  %{python_module setuptools}
-BuildRequires:  fdupes
-BuildArch:      noarch
-Requires:       python-Jinja2
-Requires:       python-PyYAML
-Requires:       python-docopt
-Requires:       python-schema
-Requires:       python3-kiwi >= 9.21.21
-%if %python_version_nodots < 37
-Requires:       python-iso8601
-%endif
-%if %{with libalternatives}
-Requires:       alts
-BuildRequires:  alts
-%else
-Requires(post): update-alternatives
-Requires(postun):update-alternatives
-%endif
-%if "%{python_flavor}" == "python3" || "%{python_provides}" == "python3"
-Provides:       python3-kiwi-keg = %version
-Obsoletes:      python3-kiwi-keg < %version
+# ------------------------------------------------------------------
+# SLE-15-SP3 ships python-setuptools 40.x – too old for a plain
+# PEP-517 build.  Everywhere else we have >= 42.
+#
+#   new_setuptools = 1   … modern setuptools (default)
+#   new_setuptools = 0   … SLE-15-SP3 or older
+# ------------------------------------------------------------------
+%define new_setuptools 1
+%if 0%{?suse_version} <= 150300
+%define new_setuptools 0
```

Comment:
I do not really see a reason when we need to build this tool for anything older than 15 SP4. Or expressed differently older than Python 3.11 which is defined at the beginning with:
`%{?sle15_python_module_pythons}` as such this can be dropped.